src/combobox/combobox.component.ts

```diff
@@ -94,6 +95,7 @@ import { Observable } from "rxjs";
 				</div>
 				<input
 					#input
+					type="text"
```

Comment:
Looking at the `carbon-components` css it seems that the background color of the input is targeted by `role='combobox'`:
```
.bx--list-box input[role='combobox'] {
  background-color: inherit;
  min-width: 0;
}
```
Not sure if that was from an earlier version or it was changed in a later version?

Reply:
actually this is what i see using latest version:
![image](https://user-images.githubusercontent.com/35496522/86021779-633f4780-ba21-11ea-8f22-b941a764b2e9.png)

Reply:
and in react version of the combobox they use `type='text'`

Reply:
So I guess it targets both but the `type='text'` selector is only available in the latest version, do you think it would be better to use `role='combobox'` instead so that earlier versions can still take advantage of the light theme @cal-smith ?

Reply:
We _should_ be able to use both I would think. `role="combobox"` is still valid (and needed) as a aria attribute, and we _should_ be specifying `type="text"` for all generic inputs